src/ucp/wireup/select.c

```diff
@@ -1685,7 +1685,9 @@ ucp_wireup_add_rma_bw_lanes(const ucp_wireup_select_params_t *select_params,
 
     bw_info.criteria.title            = "high-bw remote memory access";
     bw_info.criteria.lane_type        = UCP_LANE_TYPE_RMA_BW;
-    bw_info.max_lanes                 = context->config.ext.max_rndv_lanes;
+    bw_info.max_lanes                 = context->config.ext.proto_enable ?
+                                        UCP_MAX_LANES :
+                                        context->config.ext.max_rndv_lanes;
```

Comment:
this may create too many uct endpoints (lanes)
let's at least filter out those who lave low bandwidth compared to others (according to the configured threshold)?
going forward, we can connect the slow-path lanes on demand and not immediately during ucp endpoint creation.

Reply:
looks like it may break wire compatibility

Reply:
That's correct but for now we decided to continue with this solution, because this task is a bit urgent.
Will be fixed later on.

Reply:
imo, there is no easy way to fix it

Reply:
maybe we could pass some flag in the address to indicate max lanes?
or connect some lanes later on if there is a difference between client/server (handle "reconfig") - but that's not simple ..